Code/GraphMol/MolStandardize/Wrap/Tautomer.cpp

```diff
@@ -268,6 +268,18 @@ PyTautomerEnumeratorResult *enumerateHelper(
   return new PyTautomerEnumeratorResult(self.enumerate(mol));
 }
 
+std::vector<MolStandardize::TautomerScoringFunctions::SubstructTerm> GetDefaultTautomerSubstructsHelper() {
+  std::vector<MolStandardize::TautomerScoringFunctions::SubstructTerm> terms;
+  for(auto term: MolStandardize::TautomerScoringFunctions::getDefaultTautomerSubstructs()) {
+    terms.emplace_back(term);
+  }
+  return terms;
+}
+
+int ScoreSubstructsHelper(const ROMol &mol) {
```

Comment:
why do you need this function?

Reply:
> The substructure terms are created on a call to getDefaultSubstructTerms not on library load since they are local to a function. It’s one of the reasons I like this pattern, thread safe initialization on demand.
> 

Yeah, I had that wrong. I thought all statics were initialized at launch, this is way more useful.